### ISSUE TITLE
Missing features for MathOptAI paper

### ISSUE THREAD

**Issue by odow** [lanl-ansi/MathOptAI.jl#180]
Before publishing, we should add:

 - [ ] `nn.Conv2D`
 - [ ] `nn.MaxPool2D`
 - [x] Gradient boosted trees
 - [x] Random forrest

**Comment by odow** [comment]
Closing in favor of #66
